src/components/subscription-choices/subscription-choices.scss

```diff
@@ -0,0 +1,90 @@
+@use '../../compiled/tokens/scss/breakpoint';
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/a11y';
+@use '../../mixins/theme';
+
+///
+/// Theme properties
+///
+
+// Light theme
+@include theme.props {
+  --theme-color-background-subscription-form: #{color.$base-white};
+}
+
+// Dark theme
+@include theme.props(dark) {
+  --theme-color-background-subscription-form: #{color.$brand-primary};
+}
+
+// Alternate theme
+@include theme.props(alternate) {
+  --theme-color-background-subscription-form: #{color.$base-gray-lighter};
+}
+
+///
+/// Subscription Choices
+///
+
+.c-subscription-choices {
+  position: relative;
+}
+
+.c-subscription-choices__controls-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+  margin: size.$rhythm-default 0;
+
+  .c-button-swap {
+    flex-grow: 1;
+  }
+
+  .c-button-swap__subscribe {
+    display: flex;
+  }
+
+  @media (min-width: breakpoint.$s) {
+    flex-direction: row;
+  }
+}
+
+.c-subscription-choices__control {
+  flex-grow: 1;
+}
+
+///
+/// 1. Ensures the form background color matches the current theme
+///
+
+.c-subscription-choices__form {
+  background-color: var(--theme-color-background-subscription-form); // 1
+  block-size: 100%;
+  inline-size: 100%;
+  inset-block-start: 0;
+  inset-inline-start: 0;
+  position: absolute;
+
+  ///
+  /// 1. Allows for the visually hidden form to be keyboard accessible, will be
+  ///    visually hidden when elements within the form are not in focus.
+  ///
+
+  label,
```

Comment:
Is it intentional that you're targeting a default `label` tag here?

Reply:
Oops, I should add a CSS class to the label. Good catch!

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1679/commits/d808584988c8c3e56bdaaea2fbc56c4b951bc3ed

Reply:
Technically, BEM says you should rarely, if ever, target an element. Instead, set a class. This serves two purposes: First, it keeps all selectors on the same specificity level, and second, the BEM class names are self-documenting, so it's more clear what is happening to someone reading the markup.

`<label>` is generic. `<label class="subscription-form__label">` makes it clear what BEM module this is part of and where I can find the CSS.